css/bootstrap.min.css

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
Is this needed?

Reply:
Bootstrap is being used for the grid layout of the columns, otherwise, it can be taken out.